go-controller/pkg/config/config.go

```diff
@@ -2311,10 +2320,18 @@ func buildOvnKubeNodeConfig(ctx *cli.Context, cli, file *config) error {
 	if OvnKubeNode.Mode != types.NodeModeFull && HybridOverlay.Enabled {
 		return fmt.Errorf("hybrid overlay is not supported with ovnkube-node mode %s", OvnKubeNode.Mode)
 	}
+
+	// Warn the user if both MgmtPortNetdev and MgmtPortDPResourceName are specified since they
+	// configure the management port.
+	if OvnKubeNode.MgmtPortNetdev != "" && OvnKubeNode.MgmtPortDPResourceName != "" {
+		klog.Warningf("ovnkube-node-mgmt-port-netdev (%s) and ovnkube-node-mgmt-port-dp-resource-name (%s) "+
+			"are both specified", OvnKubeNode.MgmtPortNetdev, OvnKubeNode.MgmtPortDPResourceName)
```

Comment:
Remove `are`.
Also, from message and parameter description above not clear behaviour when both specified. And what happen, `ovnkube-node-mgmt-port-netdev` overwrites `ovnkube-node-mgmt-port-dp-resource-name`? I think it should be opposite. As had mentioned earlier, for bypass port, when resource name is specified, then it overwrites gateway interface argument. So mgmt port and bypass port arguments have inconsistent behaviour.

Reply:
Done.